src/Styleguide/Artwork/Banner.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import styled from "styled-components"
+import { Col, Row } from "../Elements/Grid"
+
+const Flex = styled.div`
+  display: flex;
+  flex-direction: row;
+`
+
+interface RoundedImageProps {
+  size: string
+}
+
+const RoundedImage = styled.img.attrs<RoundedImageProps>({})`
+  width: ${(props: RoundedImageProps) => props.size};
+  height: ${props => props.size};
+  border-radius: ${props => props.size};
+`
+
+export class Banner extends React.Component {
+  render() {
+    return (
+      <Row>
+        <Col sm={8}>
```

Comment:
@zephraph - curious about your take on this, but if possible we should keep all Row / Column dependencies out of components so that they're pure, and can be used in a variety of environments. Seems like these elements should be hoisted up to the page level.

Reply:
Yeah, I think that makes a lot of sense.